package-lock.json

```diff
@@ -2,10 +2,138 @@
   "requires": true,
   "lockfileVersion": 1,
   "dependencies": {
```

Comment:
This is weird - we have `package-lock.json` in root folder (we should only have in `backend/` & `client/`). Looks like it was removed and then added back (I guess accidentally). Anyways I would remove this change from the PR.